src/cmps/tags/Tags.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import './Tags.css';
+import Tag from '../tag/Tag';
+// eslint-disable-next-line
+import { string } from "prop-types";
```

Comment:
mmm I think this line was added by mistake
wdyt?

Reply:
Not already relevant